data/deep missions.txt

```diff
@@ -339,25 +339,29 @@ mission "Deep: Mystery Cubes 0"
 			
 			branch interested
 				has "deep: interested in cubes"
-			label interested
+			
 			choice
 				`	"Is there anything else that you need me to do?"`
 					goto next
 				`	"What is it that those cubes are for?"`
-					goto cubes
+					goto cube_answer
 			
 			label interested
 			choice
 				`	"Is there anything else that you need me to do?"`
+					goto next
 				`	"Are you able to tell me what the cubes are for now?"`
-					goto cubes
+			
+			label cube_answer
+			apply
+				set "deep: interested in cubes"
```

Comment:
just in case someone wants to build lore further, asking about the cubes at either time will set this condition